### DISCUSSION TITLE
Build time react

### DISCUSSION THREAD

**Discussion by LaCk-yt** [coollabsio/coolify#3521] [Q&A]
Hello, im testing coolify in one VPS because i really liked the idea, I have a question, why the build time for react apps takes a lot of time. 

![Screenshot 2024-09-20 at 12 14 44 PM](https://github.com/user-attachments/assets/f2553f67-3c6b-4f02-8dcb-4f348f92ba3c)
why it takes so much time in building the app, when I run npm run build, it takes 30 seconds to build it. I know that coolify is using docker (I think) and Nixpacks, but there is any way to lower this build time?

**Comment by ck-euan** [comment]
Yeah I notice similar results, builds take about 10 minutes for my app using a standalone build server
